resources/instance_analytics/collections/vG58R8k-QddHWA7_47umn_usage_analytics/cards/-fFVxT-GLz6WO41bvw-Ar_dashboards_without_recent_views.yaml

```diff
@@ -205,7 +205,6 @@ result_metadata:
   visibility_type: normal
 - base_type: type/Float
   display_name: Days since last view
-  expression_name: Days since last view
```

Comment:
I assume this file should be re-exported from somewhere rather than hand edited, but wanted to include it in the diff to have a place to hang this comment so I wouldn't forget about it.

Reply:
These are re-exported from a cloud instance but it can be a pain if there are concurrent changes. This hand edit is fine.